Automattic/wp-calypso
# Use New Common "G Suite New User List" Component in G Suite User Management

#### Changes proposed in this Pull Request

* Use New Common "G Suite New User List" Component in G Suite User Management

#### Testing instructions

1. Navigate to `/:domain/gsuite/add-users/:site` for a site and domain with G Suite
1. Fill in a couple of users in the form
1. Verify the "continue" button not work until all fields are valid
1. Verify that after clicking continue the users in the cart are correct
